src/Components/Screen.jsx

```diff
@@ -7,11 +7,18 @@ function Screen(props) {
   if (Array.isArray(text)) {
     return (
       <div className="DivTextArea" aria-label="Text Area" id="textbox1_freetext" role="textbox" tabIndex="0">
-        {text.map((item, index) => (
-          <span key={index} style={item.style}>
-            {item.char}
-          </span>
-        ))}
+        {text.map((item, index) => {
+          switch (item.char) {
```

Comment:
Hi @chavi362, no issues.
The virtual-keyboard is a good project and i like the code which is pretty good and clean.
I like to do some contribution from my side also when I am free and glad you like the code comment I made, thanks for that.
For now no need to create a new PR, will use your code. In future if you feel to optimize the code, just create a refactor branch and we can use that to optimize the code structure and logics too.
Thanks,
Kishore

Reply:
@kishoreravi24 , yes,, I would like to make a refactor branch to optimize  the code.
Do you want to do it so when I merge the refactor branch it will be your PR?
 Or you want I do it?
Thank you for your kindness
Chavi

Reply:
@chavi362 , sure will do it and share the PR with you soon!
Thanks